instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/snsattributes.go

```diff
@@ -0,0 +1,61 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otelaws // import "go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-sdk-go-v2/otelaws"
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/sns"
+	"github.com/aws/smithy-go/middleware"
+
+	"go.opentelemetry.io/otel/attribute"
+	semconv "go.opentelemetry.io/otel/semconv/v1.17.0"
+)
+
+// SNSAttributeSetter sets SNS specific attributes depending on the SNS operation being performed.
+func SNSAttributeSetter(ctx context.Context, in middleware.InitializeInput) []attribute.KeyValue {
+	snsAttributes := []attribute.KeyValue{semconv.MessagingSystem("AmazonSNS")}
+
+	switch v := in.Parameters.(type) {
+	case *sns.PublishInput:
+		var value string
+		valuePointer := v.TargetArn
+		if valuePointer == nil {
+			valuePointer = v.TopicArn
+		}
+		if valuePointer == nil {
+			if v.PhoneNumber != nil {
+				value = *v.PhoneNumber
```

Comment:
A phone number is PII data. Could we have an option to disable setting this value?

Reply:
I guess a close equivalent could be the mongo command attribute, which can be disabled:
https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/config.go#L71-L78